ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-authorization.j2

```diff
@@ -12,6 +12,16 @@
 
 {%     if aaa_authorization.config_commands is defined and aaa_authorization.config_commands == true %}
```

Comment:
This is the old config. But please change this line to:
``` if aaa_authorization.config_commands is arista.avd.defined(true)```

We will be using arista.avd.defined test to check if a knob is **defined** and is **not null**.